fbw-a380x/src/systems/instruments/src/PFD/FMA.tsx

```diff
@@ -69,12 +71,62 @@ export class FMA extends DisplayComponent<{ bus: EventBus; isAttExcessive: Subsc
 
   private AB3Message = Subject.create(false);
 
+  private readonly radioHeightConsumer = ConsumerSubject.create(this.sub.on('chosenRa'), Arinc429Word.empty());
+
+  private readonly altitudeConsumer = ConsumerSubject.create(this.sub.on('altitudeAr'), Arinc429Word.empty());
+
+  private readonly landingElevation = ConsumerSubject.create(
+    this.sub.on('landingElevation').whenChanged(),
+    Arinc429Word.empty(),
+  );
+
+  private readonly ap1Consumer = ConsumerSubject.create(this.sub.on('ap1Active').whenChanged(), false);
+
+  private readonly ap2Consumer = ConsumerSubject.create(this.sub.on('ap2Active').whenChanged(), false);
+
+  private readonly verticalModeConsumer = ConsumerSubject.create(this.sub.on('activeVerticalMode').whenChanged(), 0);
+
+  private readonly selectedVerticalSpeed = ConsumerSubject.create(this.sub.on('apVsSelected').whenChanged(), null);
+
+  private readonly selectedFpa = ConsumerSubject.create(this.sub.on('selectedFpa').whenChanged(), null);
+
+  private readonly approachCapabilityConsumer = ConsumerSubject.create(
+    this.sub.on('approachCapability').whenChanged(),
+    0,
+  );
```

Comment:
.whenChanged is redundant, and we don't do Hungarian notation (that's the part I request mandatory changes for). I suggest some names like the following (also for additional clarity). Now you have to consider if baro corrected altitude is always the right one... what if we're in STD mode?
```suggestion
  private readonly radioHeight = ConsumerSubject.create(this.sub.on('chosenRa'), Arinc429Word.empty());

  private readonly baroCorrectedAltitude = ConsumerSubject.create(this.sub.on('altitudeAr'), Arinc429Word.empty());

  private readonly landingElevation = ConsumerSubject.create(
    this.sub.on('landingElevation'),
    Arinc429Word.empty(),
  );

  private readonly ap1Active = ConsumerSubject.create(this.sub.on('ap1Active'), false);

  private readonly ap2Active = ConsumerSubject.create(this.sub.on('ap2Active'), false);

  private readonly verticalMode = ConsumerSubject.create(this.sub.on('activeVerticalMode'), 0);

  private readonly selectedVerticalSpeed = ConsumerSubject.create(this.sub.on('apVsSelected'), null);

  private readonly selectedFpa = ConsumerSubject.create(this.sub.on('selectedFpa'), null);

  private readonly approachCapability = ConsumerSubject.create(
    this.sub.on('approachCapability'),
    0,
  );
```

Reply:
Thanks for the review, I have applied your suggestion commit. Both issues raised are noted for the future 👍